specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1949,12 +1959,13 @@
                 },
                 "allowedSourceAddressPrefix": {
                     "type": "string",
-                    "description": "The IP address range that the port was opened for"
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""

```

Comment:
Some grammar and typo corrections:
`"Mutually exclusive with the \"allowedSourceAddressPrefixes\" parameter. Should be an IP address or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\"." `

Reply:
Resolved